src/scripts/deploy.js

```diff
@@ -1,56 +1,90 @@
-const assert = require('assert')
-const { series } = require('async')
-const { default: chalk } = require('chalk')
-const { exec } = require('child_process')
-const { confirm } = require('./utils')
+const assert = require("assert");
+const { series } = require("async");
+const { default: chalk } = require("chalk");
+const { exec } = require("child_process");
+const { confirm } = require("./utils");
 
-require('dotenv').config()
+require("dotenv").config();
 
-const NETWORK = process.env.NETWORK
-assert(NETWORK, 'NETWORK env var is required')
+const NETWORK = process.env.NETWORK;
+assert(NETWORK, "NETWORK env var is required");
 
-const SUBGRAPH = process.env.SUBGRAPH
-assert(SUBGRAPH, 'SUBGRAPH env var is required')
+const SUBGRAPH = process.env.SUBGRAPH;
+assert(SUBGRAPH, "SUBGRAPH env var is required");
 
-console.log(`Prepare subgraph config for network ${NETWORK}`)
-console.log(`  Generate subgraph.yaml`)
+const AUTH_TOKEN = process.env.AUTH_TOKEN;
+assert(AUTH_TOKEN, "AUTH_TOKEN env var is required");
 
-series([
-  (callback) => exec(`mustache config/${NETWORK}.json subgraph.yaml.mustache > subgraph.yaml`, null, callback),
+const VERSION = process.env.VERSION;
+assert(VERSION, "VERSION env var is required");
```

Comment:
If we add envs, shouldb't we document in the README?

Reply:
On it